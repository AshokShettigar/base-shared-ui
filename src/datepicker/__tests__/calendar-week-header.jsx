import { shallow } from 'enzyme';
import CalendarWeekHeader from 'platform-common-ui/datepicker/calendar-week-header';

describe('datepicker/calendar-week-header', function() {
    it('should render the component', function() {
        const wrapper = shallow(<CalendarWeekHeader />);
        expect(wrapper).toMatchSnapshot();
    });
});
