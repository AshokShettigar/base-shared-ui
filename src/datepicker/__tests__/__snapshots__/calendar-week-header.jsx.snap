// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`datepicker/calendar-week-header should render the component 1`] = `
<div
  className="cn-calendar-week-header"
  data-component="calendar-week-header"
>
  <div
    className="cn-calendar__column"
  >
    Sun
  </div>
  <div
    className="cn-calendar__column"
  >
    Mon
  </div>
  <div
    className="cn-calendar__column"
  >
    Tue
  </div>
  <div
    className="cn-calendar__column"
  >
    Wed
  </div>
  <div
    className="cn-calendar__column"
  >
    Thu
  </div>
  <div
    className="cn-calendar__column"
  >
    Fri
  </div>
  <div
    className="cn-calendar__column"
  >
    Sat
  </div>
</div>
`;
